website/docs/getting-started/for-designers.md

```diff
@@ -176,10 +176,11 @@ Consider using [Atomic Design](https://bradfrost.com/blog/post/atomic-web-design
 
 ### Helios Figma libraries
 
-HashiCorp product teams can use the Helios libraries published under the [HDS Design System UI Kit](https://www.figma.com/files/team/1030156573400567478) team within Figma.
+HashiCorp product teams can use the Helios libraries published under the [HDS Design System UI Kit](https://www.figma.com/files/817445202777874373/team/1030156573400567478) team within Figma.
```

Comment:
[question] instead of removing the old libraries, should we leave them but highlight that they are now deprecated? technically speaking, they're still "valid", no?

Reply:
They will be unpublished so not removed but also shouldn't appear in any new work (ideally) so we could have a note here but I personally think the clean break is nice since don't want consumers using 1.0. Similarly, we don't have any links to Structure here